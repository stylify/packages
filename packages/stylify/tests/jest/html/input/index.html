<!-- stylify-pregenerate: font-size:123px line-height:123px color:#fff color:#888  -->
<!-- stylify-pregenerate: font-size:456px line-height:456px color:yellow  -->

<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Document</title>
</head>
<body class="font-family:arial font-size:12px">
	<div class="
		text-align:center
		font-size:48px
		background:steelblue
		max-width:320px
		margin:0__auto
		color:#fff"
	>
		Index Page
	</div>
</body>
</html>
