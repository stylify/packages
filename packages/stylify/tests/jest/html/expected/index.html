<!-- stylify-pregenerate: font-size:123px line-height:123px color:#fff color:#888  -->
<!-- stylify-pregenerate: font-size:456px line-height:456px color:yellow  -->

<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Document</title>
</head>
<body class="s0 s1">
	<div class="
		s2
		s3
		s4
		s5
		s6
		s7"
	>
		Index Page
	</div>
</body>
</html>
